### PR TITLE
Update index page and Navbar

### PR DESCRIPTION
**Pull request by MichaelLally**
Updated index page and navbar.

- Motorsports logo added and now links to index page
    - Home page button removed, replaced with above
- Default .grafana-button style reworked, making default buttons colorless
    - Grey button extension now replaces old default color/highlighting
- Index page descriptions added (placeholder text)
- Grafana text box style added

### CODE REVIEW

**Comment by de846** [comment]
Simulator page buttons still look weird. The green ones are smaller than the red ones, and they cut off the text.
image

File: mercury/templates/index.html [hunk 1]
```diff
@@ -1,51 +1,69 @@
...
-      <button onclick="RedirectCAN();" class="grafana-btn grafana-btn-red" style="width: 50%; height: 40px; font-size: larger;">CAN Decoder</button>
-      <div style="height: 4px;"></div>
-      <button onclick="RedirectStopwatch();" class="grafana-btn grafana-btn-red" style="width: 50%; height: 40px; font-size: larger;">Stopwatch</button>
-      <br><br>
+    <div id="grafana-base">
+      <div id="grafana-panel">
+
+        <h1>NYU Motorsports Telemetry</h1>
+        <br>
+        This platform allows for monitoring of NYU’s Motorsports. Our simulator generates data such as suspension, temperature, acceleration, wheel speed, and fuel level. The Raw Telemetry section shows the data simulated in tabular format, while the Dashboard shows the data in easy-to-view graphs. Test CAN API by sending messages to the decoder endpoint. Access to the stopwatch to time races.
```
Comment (de846) [inline mercury/templates/index.html]: I thought we were going to have real descriptions for the site?

File: mercury/templates/index.html [hunk 2]
```diff
@@ -1,51 +1,69 @@
...
+      function RedirectCAN() {
+        window.location="/can/";
+      }
+      function RedirectStopwatch() {
+        window.location="/stopwatch/";
+      }
+      function newGrafanaWindow() {
+        window.open("https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-motorsports-telemetry-dashboard?orgId=1&refresh=5s");
+      }
+    </script>
```
Comment (de846) [inline mercury/templates/index.html]: I don't think any of this is required any more.

File: mercury/templates/index.html (hunk 1 above)
Comment (MichaelLally) [reply mercury/templates/index.html]: Working on more detailed descriptions now. Wanted to get the PR in so everything else could be reviewed.

File: mercury/templates/index.html (hunk 2 above)
Comment (MichaelLally) [reply mercury/templates/index.html]: Agreed, removing now.

Comment (de846) [reply mercury/templates/index.html]: Oops, sorry. I resolved a conflict in the GitHub UI here and just removed it. You'll need to `git pull` on this simulatorcontent branch to pick up the change.

**Comment by de846** [comment]
Oops, sorry. I made a change in the GitHub UI here to resolve a conflict for you and removed them. You'll need to `git pull` to pull down the change to sync back up.

**Comment by de846** [comment]
So I don't think this branch has all the latest changes - I just tested the CAN Decoder, and it looks weird - not current.

**Comment by de846** [comment]
The logo is pretty illegible on my screen :(
image

File: mercury/templates/index.html (hunk 2 above)
Comment (MichaelLally) [reply mercury/templates/index.html]: just updated

**Comment by MichaelLally** [comment]
I agree that the logo is hard to see, could make a lighter version but don't have time at the moment.

File: mercury/templates/index.html [hunk 3]
```diff
@@ -1,46 +1,60 @@
 {% load static %}
 <link rel="stylesheet" type="text/css" href="{% static 'mercury/style.css' %}">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
```
Comment (de846) [inline mercury/templates/index.html]: Is this being used anywhere?

**Comment by de846** [comment]
> I agree that the logo is hard to see, could make a lighter version but don't have time at the moment.

Okay - I think for the demo, it is not as good as the NYU one because it is harder to see, even if it is "more right" to use the Motorsports logo.

File: mercury/templates/index.html (hunk 3 above)
Comment (taratran) [reply mercury/templates/index.html]: I don't believe that is being used anywhere-- that was an older code that would've been used in the carousel.

**Comment by MichaelLally** [comment]
Fixing the background of the logo makes it much more visible

File: mercury/templates/index.html [hunk 4]
```diff
@@ -1,46 +1,60 @@
...
+
+        <div id="grafana-text-box">
+          <h3>Raw Telemety</h3>
+          This portal allows the raw data to be viewed. This is useful for debuggin and understanding the models behind the data visualization portal. Data generated inthe simulator will appear here.
+        </div>
+        <br>
+
+        <div id="grafana-text-box">
+          <h3>Simulator</h3>
+          Generates mock vehicle data. Allows the user to experiment with different data patters on the vidualization portal and raw telemetry portal.
```
Comment (de846) [inline mercury/templates/index.html]:
```suggestion
          Generates mock vehicle data. Allows the user to experiment with different data patters on the visualization portal and raw telemetry portal.
```

File: mercury/templates/index.html [hunk 5]
```diff
@@ -1,46 +1,60 @@
...
+
+        <div id="grafana-text-box">
+          <h3>Stop Watch</h3>
+          Time the race with the Stop Watch feature. This allows for overall time, as well as individual lap times, to be tracked in the user's browser.
+        </div>
+        <br>
+
+        <div id="grafana-text-box">
+          <h3>Raw Telemety</h3>
+          This portal allows the raw data to be viewed. This is useful for debuggin and understanding the models behind the data visualization portal. Data generated inthe simulator will appear here.
```
Comment (de846) [inline mercury/templates/index.html]:
```suggestion
          This portal allows the raw data to be viewed. This is useful for debugging and understanding the models behind the data visualization portal. Data generated in the simulator will appear here, as well as any data from vehicle sensors.
```

**Comment by de846** [comment]
> Fixing the background of the logo makes it much more visible

Agreed

File: mercury/templates/index.html (hunk 3 above)
Comment (de846) [inline mercury/templates/index.html]:
```suggestion
```

**Pull request merged**